go-app-ussd_covid19_triage.js

```diff
@@ -364,31 +632,77 @@ go.app = (function () {
           new Choice("no", $("NO")),
           new Choice("not_sure", $("NOT SURE")),
         ],
-        next: "state_tracing"
+        next: "state_age_years"
       });
     });
 
-    self.add("state_tracing", function (name) {
+    self.add("state_exposure", function (name) {
       return new ChoiceState(name, {
         question: $([
-          "Please confirm that the information you shared is correct & that the National " +
-          "Department of Health can contact you if necessary?",
+          "Have you been in close contact to someone confirmed to be infected with COVID19?",
```

Comment:
@rudigiesler should this question not be skipped for the confirmed contact case?

Reply:
Yes it is. For confirmed contacts, the state before this one is `state_breathing`. If you look at `state_breathing`, you'll see for confirmed contacts, we set `state_exposure` to `yes`, and go to `state_taste_and_smell`. So confirmed contacts won't see `state_exposure`, and will be submitted with `state_exposure=yes`